_shared/project/pyproject.toml

```diff
@@ -60,6 +60,9 @@ omit = [
     {% if cookiecutter.get("_directory") == "pyapp" %}
     "*/{{ cookiecutter.package_name }}/app.py",
     {% endif %}
+    {% if cookiecutter.get("_directory") == "pyramid-app" and cookiecutter.get("postgres") == "yes" %}
+    "*/{{ cookiecutter.package_name }}/scripts/init_db.py",
+    {% endif %}
```

Comment:
Super ugly diff rendering here. This just adds `init_db.py` to the coverage ignore list.